### DISCUSSION TITLE
[Question] What to use instead of requests package for static web page?

### DISCUSSION THREAD

**Discussion by mghiggins** [flet-dev/flet#2742] [Q&A]
### Question

I've got a simple web app that I'm writing in flet. It delegates its calculations to a separate RESTful web service, and I'm using the requests package to hit that service. That breaks the app, though, when I try to publish this as a static web page. Is there a way to call a RESTful end point from a flet web app if I build it as a static web page?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
What do you think of this: https://pyodide.org/en/stable/usage/api/python-api/http.html ?
